README.md

```diff
@@ -1,88 +1,130 @@
+### Running the Quickstart
 
-### Using Cocoapods
+To get started with the Quickstart application follow these steps:
 
-First, you will need to have Cocoapods 1.0.0+ installed on your Mac, so go ahead and do that if you haven't already - the directions are here: [Getting Started with Cocoapods](https://guides.cocoapods.org/using/getting-started.html). If you're not sure, type `pod --version` into a command line.
+1. Open this project in XCode
 
-Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
+<img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
+
+2. Next, you need to generate a valid Access Token. Log in to the Twilio Console and go to the [Testing Tools page](https://www.twilio.com/user/account/video/dev-tools/testing-tools) shown below. Pick a name for your user identity (such as "Alice"), type it in the identity textbox and click "Generate Access Token". Copy the Access Token string that will appear in the gray box - we will need in the following step.
+
+
+<img width="700px" src="images/quickstart/generate_access_tokens.png"/>
+
+*Note: If you enter the Room Name, then you can retrict this user's access to the specified Room only. Ideally, you want to implement and deploy an Access Token server to generate tokens. You can read more about setting up your own Access Token Server in this [section](#setup-an-access-token-server). Read this [tutorial](https://www.twilio.com/docs/api/video/user-identity-access-tokens) to learn more about Access Tokens.*
+
+3. Paste the token you generated in the earlier step in the `ViewController.swift`.
+
+<img width="700px" src="images/quickstart/xcode-video-quickstart-token.png"/>
 
-> NOTE: If you've used [Twilio's Cocoapods Spec Repo](https://github.com/twilio/cocoapod-specs) before you may need to update it by running `pod repo update twilio` before installing dependencies.
 
-## Access Tokens and Servers
+4. Run the Quickstart app on your iOS device or iOS Simulator. 
 
-Using Twilio's Video client within your applications requires an access token. These access tokens can come from one of two places:
+<img width="500px" src="images/emulator/emulator-iOS-video-quickstart.png"/>
 
-1) You can create a one-time use access token for testing in the [Twilio Console](https://www.twilio.com/console/video/profiles). This access token can be hard-coded directly into your mobile app, and you won't need to run your own server. This works for local development and testing, but isn't a solution for a production environment.
+5. As in Step 2, generate a new Token for another identity (such as "Bob"). Copy and paste the access token into `ViewController.swift` (replacing the one you used earlier). Build and run the app on a second physical device if you have one, or the iPhone simulator.
```

Comment:
Should we add a screenshot when app are connected to a Room, with Remote and local video?